humanitec-resource-defs/route53/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "prefix" {
+  type = string
+}
+
+variable "resource_packs_aws_rev" {
+  type = string
+}
+
+variable "access_key" {
```

Comment:
This will not work with the upcomming temporary credentials.
AWS includes a Session Token for temporary credentails

Reply:
None of the existing resources support temporary credentials yet, so I wouldn’t block this PR for this.

Happy to update all of them once the design is finalized.

Reply:
They are being added now e.g. for the route53 driver.

Reply:
Sorry I meant none of the existing pack resources support temporary credentials yet and I would generally track supporting this as a follow up.